app/uk/gov/hmrc/teamsandrepositories/connectors/ServiceConfigsConnector.scala

```diff
@@ -26,20 +28,32 @@ import scala.concurrent.{ExecutionContext, Future}
 class ServiceConfigsConnector @Inject()(
   servicesConfig: ServicesConfig,
   httpClientV2  : HttpClientV2
-){
+)(implicit ec: ExecutionContext) {
```

Comment:
The Schedulers are using their own `ExecutionContext`, not the injected one.
We previously had a problem with the schedulers consuming all the available threads, affecting the Controllers.